src/loaders/MatrixZarrLoader.js

```diff
@@ -1,18 +1,18 @@
 import { openArray } from 'zarr';
```

Comment:
Somewhat unrelatedly, while looking at the network requests, I noticed that two `.zarray` `fetch` calls are made per `openArray` - is this to be expected?

Reply:
This is because `openArray` checks that the zarr node is an array first (e.g. `.zarray` in the the store): 

https://github.com/gzuidhof/zarr.js/blob/9f42bf914ff590b1dae5455c96b021efd753171a/src/creation.ts#L139

Reply:
Gotcha, that makes sense.